test/coverage.ps1

```diff
@@ -0,0 +1,36 @@
+param(
+    [string][Alias('c')]$configuration = "Release",
+    [string][Alias('o')]$output = "CoverageOutput"
+)
+
+function Install-Tools {
+    dotnet tool restore
+}
+
+function Print-Environment {
+    dotnet --version
+    dotnet tool run coverlet --version
+}
+
+function Prepare-Output {
+    if (Test-Path -Path $output) {
+        Remove-Item $output -Recurse
+    }
+}
+
+function Run-Tests {
+    $testProjects = Get-ChildItem -Filter "*.Test.csproj" -Recurse
+
+    foreach ($testProject in $testProjects)
+    {
+        dotnet test $testProject.FullName /p:CoverletOutputFormatter="cobertura" --collect:"XPlat Code Coverage" --results-directory:"$output" -c $configuration
+    }
+
+    dotnet tool run reportgenerator -reports:$output/**/*.cobertura.xml -targetdir:$output -reporttypes:"Html;Cobertura"
```

Comment:
I would like `lcov` added to the output types since I use [Coverage Gutters](https://github.com/ryanluker/vscode-coverage-gutters) vs code extension.

It also seems like you favor the `html` output. If no one uses `cobertura`, I'd just as soon drop it from the joined output.

Reply:
That can absolutely be changed, I hadn't fully explored the world of coverage extensions for VS Code, but that one looks really nice. I have `html` and `cobertura` in there because this is a slightly modified version of something I have in my CI/CD pipeline at work using Azure DevOps. The `cobertura` file is so Azure DevOps can report on the overall percentage and the html is so manager can view it nice and pretty right from the browser. But since neither of those are applicable here I think I might make it an optional parameter to the script with the default being `lcov`.